.github/workflows/push.yml

```diff
@@ -0,0 +1,115 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+# https://github.com/actions/setup-ruby
+# https://github.com/ruby/setup-ruby#versioning
```

Comment:
We trust the verified creator [ruby](https://github.com/ruby) so I don't know if this is really telling us anything useful. The "official" `actions/setup-ruby` action [was deprecated](https://github.com/actions/setup-ruby) in favour of the `ruby/setup-ruby` one maintained by the official Ruby organization.